test/unit/region.spec.js

```diff
@@ -901,18 +901,15 @@ describe('region', function() {
         })
       });
 
-      this.MyView = Backbone.View.extend({
-        render: function() {
-          $(this.el).html('some content');
-        },
-
-        destroy: function() {}
+      this.MyView = Backbone.Marionette.View.extend({
```

Comment:
I don't think region needs to worry about what things might be called within `View.destroy` if we want Region's unit tests to make sure we don't call `View.destroy` multiple times that makes sense to me, but what happens after that points seems likes something that should be tested on the view